types/chrome/index.d.ts

```diff
@@ -5412,13 +5412,13 @@ declare namespace chrome {
          * @param details This parameter is currently unused.
          * @return The `getDefaultFontSize` method provides its result via callback or returned as a `Promise` (MV3 only).
          */
-        export function getDefaultFontSize(details?: Object): Promise<FontSizeDetails>;
+        export function getDefaultFontSize(details?: object): Promise<FontSizeDetails>;
```

Comment:
The unused details objects in `fontSettings` should probably be `unknown` for the time being, while `Object` does serve the same purpose, `unknown` is more understandable imo.